src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -535,34 +535,32 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return MakeEndOfLineTrivia(GetNextChar)
         End Function
 
+        Private Function TryGet(num As Integer, ByRef ch As Char) As Boolean
+            Dim ok = CanGet(num)
+            ch = If(ok, Peek(num), ch)
+            Return ok
+        End Function
+
         Private Function ScanLineContinuation(tList As SyntaxListBuilder) As Boolean
-            If Not CanGet() Then
+            Dim ch As Char = ChrW(0)
+            If Not TryGet(0, ch) Then
                 Return False
             End If
 
             If Not IsAfterWhitespace() Then
                 Return False
             End If
 
-            Dim ch As Char = Peek()
             If Not IsUnderscore(ch) Then
                 Return False
             End If
 
-            Dim Here = 1
-            While CanGet(Here)
-                ch = Peek(Here)
-                If IsWhitespace(ch) Then
-                    Here += 1
-                Else
-                    Exit While
-                End If
-            End While
+            Dim Here = GetWhitespaceLength(1)
+            TryGet(Here, ch)
 
-            ' Line continuation is valid at the end of the
-            ' line or at the end of file only.
-            Dim atNewLine = IsNewLine(ch)
-            If Not atNewLine AndAlso CanGet(Here) Then
+            Dim foundComment = CanGet(Here) AndAlso IsSingleQuote(Peek(Here))
```

Comment:
>CanGet(Here) AndAlso IsSingleQuote(Peek(Here)) [](start = 31, length = 46)

It looks like we should be able to use results of the ```TryGet(Here, ch)``` call above instead of redoing the work again.